src/Shipment/TaxIdentificationNumbers/TaxIdentificationNumber.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers;
+
+use function array_filter;
+
+class TaxIdentificationNumber
+{
+    public function __construct(
+        private string $countryCode,
+        private string $type,
```

Comment:
To help people choose the correct type we can actually restrict them in their options. Lets use an enum for $type. Here is some inspiration: https://github.com/MyParcelCOM/integration-commons/blob/master/src/Shipment/Customs/Incoterm.php